README.md

```diff
@@ -64,8 +64,14 @@ The `version` and `title` are mandatory fields. By default the `version` will be
 and the `title` will be `<enter your title>` if you do not provide `swagger_info/0`
 function.
 
-See the [swaggerObject specification](http://swagger.io/specification/#swaggerObject) for details
-of other information that can be included.
+See the [OpenAPI Object specification](https://swagger.io/specification/#oasObject) for details
```

Comment:
This links to the swagger 3.0 specification, it looks like the 2.0 spec has been moved to here: `https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#swagger-object`